.zshrc

```diff
@@ -1,3 +1,6 @@
+# WARNING: `pyenv init -` no longer sets PATH.
+eval "$(pyenv init -)"
```

Comment:
The `.profile` should load in either shell, so I don't think this is actually necessary.

Reply:
the pyenv docs say this is to enable autocompletion and subcommands